app/uk/gov/hmrc/gform/views/form/addToList.scala.html

```diff
@@ -79,8 +77,9 @@
 
     @pageLevelError.render
 
-    @hmrcPageHeading(PageHeading(text = heading))
-
+    <h1 class="govuk-heading-l">
```

Comment:
Also let's use our own `pageHeading.scala.html` for consistency as we do elsewhere instead of custom html.

Reply:
Done